server/src/main/java/com/lunchtime/mapper/PersonMapper.java

```diff
@@ -26,6 +26,7 @@ public Person fromDtoToPerson(PersonDto personDto) {
             person.setPhoneNumber(personDto.getPhoneNumber());
             person.setEmail(personDto.getEmail());
             person.setPhotoUrl(personDto.getPhotoUrl());
+            person.setRoleId(personDto.getRole());
             return person;
```

Comment:
Nothing serious, but we should name setters exactly what we mean. I'd prefer `roleId` field, as I did in my task

![image](https://user-images.githubusercontent.com/50665431/80914932-8d44f800-8d57-11ea-9421-8b34dd1d9d69.png)

Reply:
you're right, fixed)